pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/operator/AggregateOperatorTest.java

```diff
@@ -51,4 +233,27 @@ public void testGroupByAggregateWithHashCollision() {
     Assert.assertEquals(resultRows.get(0), expectedRows.get(0));
     Assert.assertEquals(resultRows.get(1), expectedRows.get(1));
   }
+
+  @Test(
+      expectedExceptions = IllegalStateException.class,
+      expectedExceptionsMessageRegExp = ".*Unexpected value: AVERAGE.*")
+  public void shouldThrowOnUnknownAggFunction() {
```

Comment:
Can we also have a test covering when the input type is not a number? I assume this will throw an exception in Java?